deploy/operator/observability-operator-service.yaml

```diff
@@ -13,3 +13,5 @@ spec:
   ports:
   - name: metrics
     port: 8080
+  - name: healthz
```

Comment:
Do we need to expose the healthz port?

Reply:
My thinking was that it had to be reachable within the cluster. Is that wrong assumption?

Reply:
the probe will only be used by the kubelet, it doesn't have to be exposed in the cluster.

Reply:
Ahh OK didn't know that. I'll remove it and then I guess it brings up the question about testability of this.